DESCRIPTION

```diff
@@ -1,11 +1,13 @@
 Package: pkgload
 Title: Simulate Package Installation and Attach
-Version: 1.4.0.9000
+Version: 1.4.0.9001
 Authors@R: c(
     person("Hadley", "Wickham", role = "aut"),
     person("Winston", "Chang", role = "aut"),
     person("Jim", "Hester", role = "aut"),
     person("Lionel", "Henry", , "lionel@posit.co", role = c("aut", "cre")),
+    person("Mauricio", "Vargas Sepulveda", role = "ctb",
+        comment = c(ORCID = "0000-0003-1017-7574")),
```

Comment:
```suggestion
```

```suggestion
```

Sorry if there was a misunderstanding, we use the author field for long time contributors or people who have contributed a significant portion of the code.